### PR TITLE
[Test-IPv6.cz] Merge and simplify rulesets

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/Test-IPv6.cz.xml [hunk 1]
```diff
@@ -1,52 +1,21 @@
...
-		<exclusion pattern="^http://(?:www\.)?test-ipv6\.cz/(?!\w+\.css|\w+\.js|images/|ipv6widget/|site/)" />
-
-
-	<securecookie host="^(?!www\.).+\.test-ipv6\.cz$" name=".+" />
+	<target host="www.test-ipv6.cz" />
+	<target host="a.test-ipv6.cz" />
+	<target host="ds.test-ipv6.cz" />
+	<target host="ipv4.test-ipv6.cz" />
+	<target host="v4.test-ipv6.cz" />
+	<target host="www4.test-ipv6.cz" />
```
Comment (J0WI) [inline src/chrome/content/rules/Test-IPv6.cz.xml]: You missed all IPv6 targets:
```
aaaa.test-ipv6.cz
ds.v6ns.test-ipv6.cz
ipv4.test-ipv6.cz
ipv6.test-ipv6.cz
mtu1280.test-ipv6.cz
v6.test-ipv6.cz
www6.test-ipv6.cz

Comment (Bisaloo) [reply src/chrome/content/rules/Test-IPv6.cz.xml]: They were failing over both `http` and `https` for me but perhaps that's due to my setup (?)

Comment (J0WI) [reply src/chrome/content/rules/Test-IPv6.cz.xml]: You probably don't have IPv6, they WFM.

Comment (J0WI) [reply src/chrome/content/rules/Test-IPv6.cz.xml]: Still missing: `ds.v6ns.test-ipv6.cz`

Comment (Bisaloo) [reply src/chrome/content/rules/Test-IPv6.cz.xml]: This subdomain doesn't exist for me (even on a computer with IPv6). I can add it if you want but I need you to tell me where. As a target?

Comment (J0WI) [reply src/chrome/content/rules/Test-IPv6.cz.xml]: It works for me on IPv4 and IPv6. Can you try to add it as target?

**Comment by J0WI** [comment]
All tests passed, Thanks!

**Pull request merged**